doc/pages/faq.asciidoc

```diff
@@ -155,36 +225,42 @@ for a given format for instance), but not much can be done about those
 ambiguous cases. You might consider writing a custom `$HOME/.magic` file
 if needed.
 
-== Can I disable syntax highlighting completely ?
+=== Can I disable syntax highlighting completely ?
 
 Similarly to the indentation hooks, the name format followed by the
 highlighting hooks is `<lang>-highlight`. You can thus disable syntax
 highlighting using the following command: `set global disabled_hooks
 '.+-highlight'`
 
-== Why does a dot `.` in a regex select newline characters ?
+== The editing language
+
+=== Why aren't there other scopes similar to `%sh{}` e.g. python ?
+
+Supporting custom scopes would add hard dependencies to the project, which
+is too much of a drawback when balanced against the low cost of using
+an interpreter in a regular shell scope (e.g. `%sh{ python -c "..." }`).
+The shell scope allows users to spawn any interpreter they want, for a minimal
+cost in terms of performance, it is therefore the reason why it's the only
+one available by default.
+
+=== What shell is used to expand `%sh{}` scopes ?
+
+The server expands shell scopes using the `sh` binary, stored in one of the
+directories where all the POSIX standard utilities can be found -this list
```

Comment:
`found -this` should probably be `found - this` (space before and after the punctuation) or `found-this` (no space)

Reply:
Thanks for the review, I've addressed the comment in a "prose fix" commit.